resources/views/landing.blade.php

```diff
@@ -18,7 +18,15 @@
     </div><!-- header -->
 
 
-    @include('includes.menu')
```

Comment:
If the include is removed, the file `resources/view/includes/menu.blade.php` can be deleted as well.

Reply:
fixed with [0fe09e8](https://github.com/LycheeOrg/Lychee/pull/1489/commits/0fe09e8dd5763b1aa49aeb017d00f1ba09e8d8ad)